### PR TITLE
Updated profile

### LINKED ISSUE #45: Profile View UI improvements to match designs

### PR DESCRIPTION
**Pull request by sacchit25**
# Profile View Updated

## Matched the Figma Design

Fixes #45 

Screenshot 2024-10-31 at 7 08 25 PM

### CODE REVIEW

File: Vineyard/Vineyard/ProfileView.swift [hunk 1]
```diff
@@ -22,37 +22,27 @@ struct ProfileView: View {
...
+                                     .font(.system(size: 16))
+                                     .fontWeight(.regular)
+                                     .foregroundColor(.gray)
                             }
                         }
                     }
                     .frame(maxWidth: .infinity, alignment: .leading)
-                    .padding(.top, 44)
+                    .padding()
+                    .background(Color(red: 0.937, green: 0.937, blue: 0.937))
```
Comment (rahulon12) [inline Vineyard/Vineyard/ProfileView.swift]: We should store hardcoded colors in an extensions file. You should create a group called "Extensions", and in that a file called "Color+Extensions":

```swift
// in Color+Extensions.swift

extension Color {
    static var profileViewCellBackground: Color {
        Color(red: 0.937, green: 0.937, blue: 0.937)
    }
}
```
and then:

```suggestion
                    .background(Color.profileViewCellBackground)
```

File: Vineyard/Vineyard/ProfileView.swift [hunk 2]
```diff
@@ -81,13 +71,14 @@ struct BadgeView: View {
...
+            Image("Vector")
+                .resizable()
+                .frame(width: 40, height: 140)
                 .padding()
-                .frame(height: 100)
-                .background(Color.gray.opacity(0.5))
-                .cornerRadius(10)
         }
+        .frame(width: 110, height: 200)
+        .background(Color(red: 0.937, green: 0.937, blue: 0.937))
```
Comment (rahulon12) [inline Vineyard/Vineyard/ProfileView.swift]: Same here:

```suggestion
        .background(Color.profileViewCellBackground)
```

File: Vineyard/Vineyard/ProfileView.swift (hunk 1 above)
Comment (rahulon12) [reply Vineyard/Vineyard/ProfileView.swift]: Also, is this the same color used in dark mode?

Comment (sacchit25) [reply Vineyard/Vineyard/ProfileView.swift]: i'm not too sure if the updated push accounts for light/dark mode, can you lmk if it does not?

**Comment by sacchit25** [comment]
Screenshot 2024-11-06 at 11 52 09 PM

File: Vineyard/Vineyard/ProfileView.swift (hunk 1 above)
Comment (rahulon12) [reply Vineyard/Vineyard/ProfileView.swift]: It won't change dynamically, but that's fine for now.

**Review by rahulon12 — APPROVED** [review]
LGTM

**Pull request merged**